docs/Radarr/Radarr-recommended-naming-scheme.md

```diff
@@ -29,75 +20,133 @@ Finally, enable `Rename Movies` to reveal the movie renaming options.
 
 ## Standard Movie Format
 
-This naming scheme is made to be compatible with the [New Plex Agent](https://forums.plex.tv/t/new-plex-media-server-movie-scanner-and-agent-preview/593269/517){:target="\_blank" rel="noopener noreferrer"} that now supports IMDb and TMDb IDs in filenames, if you don't need it or want it just remove `{imdb-{ImdbId}}`
+IMDb IDs are very accurate and rarely change. In contrast, TMDb IDs do change or are removed more frequently, but they are more aligned with Radarr.
```

Comment:
```suggestion
While both IMDb and TMDb IDs are unique, TMDb can occasionally remove IDs entirely, sometimes only to be readded with a new ID later. However, due to using TMDb as its metadata source, they can be seen as "more aligned" with Radarr. IMDb IDs on the other hand, once present, are very accurate and rarely ever change. 
```
Not super sure on this, but this seems good to me like this, thoughts? @yammes08 ?